.changeset/fast-carrots-divide.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/cms-site": minor
+---
+
+Add rel=0 to youtube search parameters
+
+By setting the parameter rel to 0 in the `YouTubeVideoBlock` only related videos from the same channel as the embedded video are shown.
```

Comment:
```suggestion
Disable showing related videos from other channels in `YouTubeVideoBlock`

By setting the parameter `rel` to `0` only related videos from the same channel as the embedded video are shown.
```

I'd change the title to _what_ you're changing, not _how_ you're achieving it (since it's an implementation detail)